### ISSUE TITLE
Unable to build Windows binaries. Are they available to download anywhere? 👀 

### ISSUE THREAD

**Issue by ThreeDeeJay** [plowteam/donut#28]
I've tried compiling forks of both the [master branch](https://github.com/ThreeDeeJay/donut/actions/runs/4033196663) and the [pure3d_manager](https://github.com/ThreeDeeJay/donut/actions/runs/4033194192) which seems newer but no luck so far.
Perhaps @handsomematt or @aylaylay happen to have them handy? 👀 
also is there still a discord server? the links are dead 💀

**Comment by Hendo16** [comment]
Yeah, I fear this is a dead project. I wonder if our only hope now is to get the leaked source code to compile for windows 10 and then build it out from there to be compiled across a wide array of platforms

**Comment by Svxy** [comment]
> I wonder if our only hope now is to get the leaked source code

[shar](https://github.com/Svxy/The-Simpsons-Hit-and-Run)

**Comment by agret** [comment]
It seems the github project is dead (he has deleted the discord server too) but people are still working on it

https://donutteam.com/blog/post/dmdu2
